src/cal.py

```diff
@@ -21,4 +21,37 @@
 
 import sys
 import calendar
-from datetime import datetime
\ No newline at end of file
+from datetime import datetime
+
+
+def check_year(year):
+    min_year = calendar.datetime.MINYEAR
+    max_year = calendar.datetime.MAXYEAR
+
+    return min_year <= int(year) <= max_year
+
+
+def check_month(month):
+    return 1 <= int(month) <= 12
```

Comment:
It's smart to add these check functions but you might be able to assume they're not necessary in a controlled environment. Good work though.